src/application/components/Queries/Queries.tsx

```diff
@@ -63,12 +63,18 @@ interface QueriesProps {
 
 export const Queries = ({ clientId, explorerIFrame }: QueriesProps) => {
   const [selected, setSelected] = useState("1");
-  const { data, startPolling, stopPolling } = useQuery(GET_QUERIES, {
-    variables: { clientId: clientId as string },
-    skip: clientId == null,
-    fetchPolicy: "cache-and-network",
-    pollInterval: 500,
-  });
+  const { loading, error, data, startPolling, stopPolling } = useQuery(
+    GET_QUERIES,
+    {
+      variables: { clientId: clientId as string },
+      skip: clientId == null,
+      pollInterval: 500,
+    }
+  );
+
+  if (error) {
+    throw error;
+  }
```

Comment:
If I add something along the lines of
```
  if (2 > 1)
    throw new ApolloError({ networkError: new Error("Network error") });
```
here, the whole extension always ends up in a "Client not found" state.

Could you look into that a bit more?

Reply:
Yeah, the `2>1` is to keep the TS compiler from yelling that a lot of code after the fixed `throw` will not be executed.

Generally, I was just throwing errors here to test these changes. What stood out for me was that doing this in the mutation panel had no effects on the connection, but doing so from within the queries panel made the whole connection impossible.
This might be perfectly okay, but it felt a bit weird and I wanted to point it out.